k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```diff
@@ -133,10 +133,10 @@ spec:
                     title: Hearing Management Interface
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*PRE-.*\/.+\/master
+                      ^HMCTS_.*PRE-.*\/(pre-shared-infrastructure)\/master
```

Comment:
any reason you don't want all apps in the folder showing up?

the previous seems right?